lib/messageFormatFactory.js

```diff
@@ -3,12 +3,13 @@ const formatDate = require('./formatDate')
 const colorizerFactory = require('pino-pretty').colorizerFactory
 
 const messageFormatFactory = (levels, colors, useColors) => {
-  const customColors =
-    colors != null
-      ? Object.entries(colors).reduce((colors, [level, color]) => {
-        return [...colors, [level, color]]
-      }, [])
-      : undefined
+  let customColors
+  if (colors !== null) {
```

Comment:
I know what I wrote, but we should verify that the more permissive check isn't required here.

Reply:
Reverted to loose inequality as tests were failing, so assumed it was intended.